app/library/api_schemas.py

```diff
@@ -19,11 +22,14 @@
 admin_scenario_post_handler_schema = {
     "type": "object",
     "properties": {
-        "name": {"type": "string", "pattern": r"^[\'a-zA-Z0-9 _-]{1,50}$"},
+        "name": {
+            "type": "string",
+            "pattern": r"^[\:\(\)\'\?\!\.\",a-zA-Z0-9 _-]{1,50}$",
```

Comment:
Should this be 2000 instead of 50 to match room description size?

![image](https://user-images.githubusercontent.com/35475783/156953146-fd29dc6f-4a58-487e-9981-4efb33d81912.png)